sklearn/ensemble/voting_classifier.py

```diff
@@ -192,14 +205,16 @@ def fit(self, X, y, sample_weight=None):
 
         transformed_y = self.le_.transform(y)
 
-        self.estimators_ = Parallel(n_jobs=self.n_jobs)(
-                delayed(_parallel_fit_estimator)(clone(clf), X, transformed_y,
-                                                 sample_weight=sample_weight)
-                for clf in clfs if clf is not None)
+        if not self.prefit:
+            self.estimators_ = Parallel(n_jobs=self.n_jobs)(
+                    delayed(_parallel_fit_estimator)(
+                        clone(clf), X, transformed_y,
+                        sample_weight=sample_weight)
+                    for clf in clfs if clf is not None)
+        else:
+            self.estimators_ = [clf for clf in clfs if clf is not None]
```

Comment:
You don't appear to have tested the `clf is None` case. Perhaps it's appropriate to test the prefit behaviour by using something like this in each of the current tests:
```python
def check_prefit_invariance(vc, X):
    """Checks that an equivalent estimator with prefit=True yields the same results"""
    prefit_vc = clone(vc)
    prefit_vc.estimators = [(name, est) for (name, _), est in zip(vc.estimators, vs.estimators_)]
    assert np.all(prefit_vc.predict(X) == vc.predict(X))
```
Then again, that may be overkill.